salt/modules/napalm_route.py

```diff
@@ -82,7 +82,7 @@ def show(destination, protocol=None, **kwargs):  # pylint: disable=unused-argume
 
     .. code-block:: bash
 
-        salt 'my_router' route.show
+        salt 'my_router' route.show 172.16.0.0/25
```

Comment:
There is no option to not pass a destination, its a required parameter. @mirceaulinic I just updated the docstring but if you can actually pass a None value the method definition should be fixed